backend/internal/flow/utils/utils.go

```diff
@@ -294,15 +296,36 @@ func getIDP(idpName string) (*idpmodel.IDP, error) {
 	idpSvc := idpservice.GetIDPService()
 	idp, err := idpSvc.GetIdentityProviderByName(idpName)
 	if err != nil {
-		return nil, fmt.Errorf("error while getting IDP with the name %s: %w", idpName, err)
-	}
-	if idp == nil {
+		if idpName == "Local" && errors.Is(err, idpmodel.ErrIDPNotFound) {
+			return getDefaultLocalIDP()
+		}
 		return nil, fmt.Errorf("IDP with name %s does not exist", idpName)
 	}
 
 	return idp, nil
 }
 
+// getDefaultLocalIDP retrieves the default Local IDP configuration.
+func getDefaultLocalIDP() (*idpmodel.IDP, error) {
+	runtimeConfig := config.GetThunderRuntime()
+	if runtimeConfig == nil {
+		return nil, fmt.Errorf("runtime configuration not available")
+	}
```

Comment:
This check is redundant as `GetThunderRuntime()` function panics when the runtimeConfig is nil